dask_sql/physical/rel/custom/predict.py

```diff
@@ -59,8 +61,21 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
 
         model, training_columns = context.schema[schema_name].models[model_name]
         df = context.sql(sql_select)
-        prediction = model.predict(df[training_columns])
-        predicted_df = df.assign(target=prediction)
+        try:
+            prediction = model.predict(df[training_columns])
+            predicted_df = df.assign(target=prediction)
+        except TypeError:
```

Comment:
In what scenarios do we hit this case?

Reply:
That makes sense. So this is specifically for the case where predict returns a numpy array instead of a Dask Array.